README.md

```diff
@@ -21,7 +21,7 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 Blog_App-Ruby <a name="about-project"></a>
+# 📖 Blog App <a name="about-project"></a>
```

Comment:
- [optional] to make your README file more professional, kindly remove the live demo section, since your project is yet to be deployed. This can be added later on. 😄